### PR TITLE
Added Axios for Api connection

### PR DESCRIPTION
**Pull request by aryaincedoer**
generated axios request for signup

### CODE REVIEW

File: src/components/Signup/Signup.js [hunk 1]
```diff
@@ -8,102 +8,177 @@ import {
...
+    addressId: null,
+  });
+
+  function submitHandler() {
+    console.log("axios", formData);
+    axios
+      .post("https://localhost:44371/api/v1/user", formData)
+      .then((response) => console.log(response));
+  }
+  function clicker(e) {
```
Comment (ShubhamIncedo1) [inline src/components/Signup/Signup.js]: Can we have action explanatory function name like onChangeHandler

File: src/components/Signup/Signup.js [hunk 2]
```diff
@@ -8,102 +8,177 @@ import {
...
+    console.log("e=", e);
```
Comment (ShubhamIncedo1) [inline src/components/Signup/Signup.js]: please remove unusual console

**Comment by aryaincedoer** [comment]
changes done

File: src/components/Signup/Signup.js (hunk 1 above)
Comment (aryaincedoer) [reply src/components/Signup/Signup.js]: changes done

**Review by ShubhamIncedo1 — APPROVED** [review]

**Pull request merged**